### ISSUE TITLE
Example in readme throw an error on MacOS

### ISSUE THREAD

**Issue by AurelienFT** [hawolt/league-client-api#2]
Hello,

Thank you for making this project, I want to look at it and understand how it works. I first tried to launch the example code. I'm very new to Java. I successfully launched it there is a lot of logs of API calls that happens so I think everything goes well until I get this error : 

```
[25 May 2023 23:22:28] [DEBUG] [entitlement] putting riot-client.entitlements_token: hidden by me
[WARNING] 
java.lang.reflect.InvocationTargetException
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:293)
    at java.lang.Thread.run (Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "com.hawolt.yaml.YamlWrapper.get(com.hawolt.yaml.ConfigValue)" because "wrapper" is null
    at com.hawolt.virtual.leagueclient.VirtualLeagueClientInstance.login (VirtualLeagueClientInstance.java:118)
    at com.demo.app.App.main (App.java:28)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:293)
    at java.lang.Thread.run (Thread.java:833)
```

I'm on MacOs 13.3

**Comment by AurelienFT** [comment]
Dived a bit through the code it seems that the project isn't made for macos at all. You can close it if you want. I wanna make something similar in Rust if you have some times to exchange on your ways to dive through the API and Config file of LoL I would be very happy !

**Comment by hawolt** [comment]
I believe that is because RMAN fails to parse the manifest and extract the file from the bundle hence why the YamlWrapper is null, it should work on mac aswell as long as RMAN works but it currently does not

**Comment by hawolt** [comment]
Should be resolved with 2630aa2912af7d6266d1f2e55fa13e3988c3105b which fixed RMAN, any chance you can check back so I can close this issue?

**Comment by AurelienFT** [comment]
I tested now I have : 
```
[18 Aug 2023 00:55:35] [INFO] Client setup complete
```

Should be fine !
